### ISSUE TITLE
Change filename patterns

### ISSUE THREAD

**Issue by eldark** [AssetKid/raider-release#13]
I suggest the possibility of altering the way files are named on download. For example, adding the username and a dash before the post attachment id or real filename (something like what JDownloader2 does).

**Comment by avekifes** [comment]
Yes, I'm itching to switch to Raider, but I'm waiting for this feature to be implemented first for the sake of consistency. The current filename pattern requires better separation between date and time of download, I think.

Raider's current pattern: 2019-10-05-03h20m.2170317690023431450

With one or two brackets, possibly: [2019-10-05] [03.20.00] 2170317690023431450

(Notice how I added seconds to the time? Can that be implemented?)
